Dockerfile

```diff
@@ -38,6 +38,8 @@ RUN echo "source /root/mambaforge/bin/activate team3_env" >> ~/.bashrc
 # Copy requirements.txt into the container
 COPY requirements.txt /app/requirements.txt
 
+RUN mamba install -c conda-forge jupyter ipykernel
+
```

Comment:
We don't need this line. You can add ipykernel in requirements.txt instead

Reply:
Okayy I will change it